app/models/client.rb

```diff
@@ -759,6 +759,536 @@ def self.sort_options(show_vispdat: false, show_assessment: false)
+        description: nil,
+        type: 'Boolean',
+      },
+      physical_disability: {
+        title: 'Physical Disability',
+        description: nil,
+        type: 'Boolean',
+      },
+      disabling_condition: {
+        title: 'Disabling Condition',
+        description: nil,
+        type: 'Boolean',
+      },
+      release_of_information: {
+        title: "#{Translation.translate('Release of information')} signed on",
+        description: nil,
+        type: 'Datetime',
+      },
+      dmh_eligible: {
+        title: 'DMH Eligible',
+        description: nil,
+        type: 'Boolean',
+        display_check: 'can_view_dmh_eligibility?',
+      },
+      va_eligible: {
+        title: 'VA Eligible',
+        description: nil,
+        type: 'Boolean',
+        display_check: 'can_view_va_eligibility?',
+      },
+      hues_eligible: {
+        title: 'Hues Eligible',
+        description: nil,
+        type: 'Boolean',
+        display_check: 'can_view_hues_eligibility?',
+      },
+      disability_verified_on: {
+        title: 'Disability Verified On',
+        description: nil,
+        type: 'Datetime',
+      },
+      income_total_monthly: {
+        title: 'Total Monthly Income',
+        description: nil,
+        type: 'Float',
+      },
+      hiv_positive: {
+        title: 'HIV+',
+        description: nil,
+        type: 'Boolean',
+        display_check: 'can_view_hiv_positive_eligibility?',
+      },
+      housing_release_status: {
+        title: 'Housing Release',
+        description: nil,
+        type: 'String',
+      },
+      vispdat_score: {
+        title: 'VI-SPDAT Score',
+        description: nil,
+        type: 'Integer',
+        display_check: 'can_view_vspdats?',
+      },
+      family_member: {
+        title: Translation.translate('Part of a family') ,
+        description: nil,
+        type: 'Boolean',
+      },
+      child_in_household: {
+        title: Translation.translate('Children under age 18 in household'),
+        description: nil,
+        type: 'Boolean',
+      },
+      lifetime_sex_offender: {
+        title: Translation.translate('Life-Time Sex Offender') ,
+        description: nil,
+        type: 'Boolean',
+      },
+      meth_production_conviction: {
+        title: Translation.translate('Meth Production Conviction'),
+        description: nil,
+        type: 'Boolean',
+      },
+      days_homeless: {
+        title: Translation.translate('Cumulative Days Homeless'),
+        description: Translation.translate('Cumulative days homeless, all-time'),
+        type: 'Integer',
+      },
+      ha_eligible: {
+        title: Translation.translate('Housing Authority Eligible'),
+        description: nil,
+        type: 'Boolean',
+      },
+      days_homeless_in_last_three_years: {
+        title: Config.using_pathways? ? 'Total # of nights' : Translation.translate('Days Homeless in Last Three Years'),
+        alt_title: 'Days homeless in the last three years',
```

Comment:
I like the consistency of always using the same title if possible. so:
```suggestion
        title: Config.using_pathways? ? 'Total # of nights' : Translation.translate('Days Homeless in Last Three Years'),
```